optimum/habana/transformers/models/llama/modeling_llama.py

```diff
@@ -748,6 +761,7 @@ def forward(
                     use_flash_attention=use_flash_attention,
                     flash_attention_recompute=flash_attention_recompute,
                     flash_attention_causal_mask=flash_attention_causal_mask,
+                    flash_attention_fast_softmax=flash_attention_fast_softmax,
                     cache_idx=cache_idx,
                     use_fused_rope=use_fused_rope,
```

Comment:
Please rebase and add argument to self._gradient_checkpointing_func too

Reply:
Done in latest commits.